SQL/New_patches/2022-12-05-AddVizConfig.sql

```diff
@@ -0,0 +1,27 @@
+-- Adds the option to toggle the EEG Browser visualization components (disabled by default).
```

Comment:
You're missing the update to the schema for new LORIS installs in the 0000-00-xxx files. And raisinbread probably needs to be updated too?

Reply:
@driusan Done